src/handlers/Registrar.ts

```diff
@@ -23,7 +23,7 @@ export const makeRegistrarHandlers = (ownedName: `${string}eth`) => {
     if (domain.labelName !== name) {
       await context.db
         .update(schema.domain, { id: node })
-        .set({ labelName: name, name: `${name}${ownedName}` });
+        .set({ labelName: name, name: `${name}.${ownedName}` });
```

Comment:
Fixing the bug introduced when we dropped the starting dot from the `ownedName` param.